security/azure/sql_publicly_accessible_managed_instance/README.md

```diff
@@ -1,62 +1,50 @@
 
-1. Follow steps to [Create an Azure Active Directory Application](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-azure-active-directory-application)
-1. Grant the Azure AD Application access to the necessary subscription(s)
-1. [Retrieve the Application ID & Authentication Key](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key)
-1. Create RightScale Credentials with values that match the Application ID (Credential name: `AZURE_APPLICATION_ID`) & Authentication Key (Credential name: `AZURE_APPLICATION_KEY`)
-1. [Retrieve your Tenant ID](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id)
+### Credential configuration
 
-### Functional Details
+For administrators [creating and managing credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) to use with this policy, the following information is needed:
 
-When a publicly accessible Azure SQL Managed Instance is detected, an email action is triggered automatically to notify the specified users of the incident. Users then have multiple actions that they are able to take after approval:
-- *delete* - deletes the Azure SQL managed instance
-- *Note: by default *delete* action has been disabled, the user can follow the steps mentioned in "To enable delete action" section above to enable delete action.*
-- *disable public data endpoint* - modifies the configuration of virtual network of the particular SQL managed instance that allows public accessibility
+Provider tag value to match this policy: `azure_rm`
 
-### Input Parameters
+## To enable delete action
 
-This policy has the following input parameters required when launching the policy.
+Perform below steps to enable delete action.
 
-- *Email addresses to notify* - Email addresses of the recipients you wish to notify when new incidents are created
-- *Exclusion Tag Key* - Azure SQL Managed instance tag to ignore instance that are with public data endpoint enabled. Only supply the tag key. The policy assumes that the tag value is irrelevant.
-- *Azure AD Tenant ID* - the Azure AD Tenant ID used for the Azure API Authentication
-- *Azure Subscription ID* - the Azure Subscription ID used for the Azure API Authentication
+- Edit the file [Check_for_publicly_accessible_Azure_SQL_Managed_Instance](https://github.com/flexera/policy_templates/tree/master/security/azure/sql_publicly_accessible_managed_instance)
+- uncomment the line which conatins 'escalate $esc_delete_Managed_instances_approval' and save the changes.
+- upload the modified file and apply the policy.
+
+## Azure Required Permissions
+
+- Microsoft.Sql/managedInstances/*
```

Comment:
Remove the Azure Required Permissions, but move the bullet under Cloud Configuration.  All of the credential and permissions should be under the Prerequisite

Reply:
Removed the Azure Required Permissions and added the bullet under Prerequisite